<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dias_rename</class>
 <widget class="QDialog" name="dias_rename">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>442</width>
    <height>554</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dias schneiden</string>
  </property>
  <widget class="QWidget" name="centralWidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>431</width>
     <height>553</height>
    </rect>
   </property>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>371</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>Anzahl Dias</string>
    </property>
    <widget class="QRadioButton" name="rb_1">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>41</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string notr="true">1</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rb36">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>50</y>
       <width>31</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string notr="true">36</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rb_50">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>50</y>
       <width>41</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string notr="true">50</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rb_80">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>50</y>
       <width>31</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string notr="true">80</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="rb_100">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>50</y>
       <width>41</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string notr="true">100</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rb_n">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>50</y>
       <width>31</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string notr="true">? :</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="sb_n">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>50</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>200</number>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Rest:</string>
     </property>
    </widget>
    <widget class="QLabel" name="la_rest">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>20</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string notr="true">0</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rb_60">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>50</y>
       <width>41</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string notr="true">60</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>391</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>Verzeichnis wählen / neu laden</string>
    </property>
    <widget class="QLabel" name="la_dir">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>271</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_dir">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>20</y>
       <width>41</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string notr="true">...</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>20</y>
       <width>41</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string notr="true">♻</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="le_us">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>371</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string>Maske</string>
    </property>
    <widget class="QLineEdit" name="le_prefix">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QSpinBox" name="sb_schiene">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>30</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_us_2">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>30</y>
       <width>21</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>_</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="sb_start">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>30</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>200</number>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>270</y>
      <width>371</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>Löschen</string>
    </property>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>lösche Bild:</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="sb_delStart">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>20</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>20</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>bis</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="sb_delEnde">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>20</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="la_delStart">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>60</y>
       <width>201</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>1. zu löschende Bild:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>letztes zu löschende Bild:</string>
     </property>
    </widget>
    <widget class="QLabel" name="la_delEnd">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>80</y>
       <width>201</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>20</y>
       <width>151</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>0-...: nichts löschen</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>390</y>
      <width>371</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string>Umbenennen</string>
    </property>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>1. umzubenennendes Bild:</string>
     </property>
    </widget>
    <widget class="QLabel" name="la_ren1q">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>30</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>30</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>→</string>
     </property>
    </widget>
    <widget class="QLabel" name="la_ren1z">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>30</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>x. umzubenennendes Bild:</string>
     </property>
    </widget>
    <widget class="QLabel" name="la_renxq">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>50</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>50</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>→</string>
     </property>
    </widget>
    <widget class="QLabel" name="la_renxz">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>50</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pB_go">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>480</y>
      <width>181</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Go!</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>480</y>
      <width>151</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Einstellungen sichern</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_Abbruch">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>520</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Abbruch</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
